c2cgeoportal/scaffolds/update/testDB/+dot+dockerignore

```diff
@@ -0,0 +1 @@
+*.mako
```

Comment:
If I add an endline, Travis is failing because there is an endline:
```
c2cgeoportal/scaffolds/update/testDB/+dot+dockerignore:2: new blank line at EOF.
```